nodedb.go

```diff
@@ -211,6 +219,10 @@ func (ndb *nodeDB) SaveFastNode(node *FastNode) error {
 }
 
 func (ndb *nodeDB) setStorageVersion(newVersion string) error {
+	if newVersion >= fastStorageVersionValue {
```

Comment:
what does >= mean for a string here

Reply:
Oh wow, >= is supported on strings in golang, I thought you had to use strings.Compare

Cool!

Reply:
Can we add a comment specifying the two expected formats of each. I'm mildly worried newVersion could be called with something not complying with a semver version

Reply:
As per [this comment](https://github.com/osmosis-labs/iavl/pull/12#discussion_r795826447), refactored this method and addressed the issue.